src/gui/askforoauthlogindialog.cpp

```diff
@@ -35,6 +35,15 @@ AskForOAuthLoginDialog::AskForOAuthLoginDialog(AccountPtr accountPtr, QWidget *p
 
     setFixedSize(this->sizeHint());
 
+    auto creds = qobject_cast<HttpCredentialsGui *>(accountPtr->credentials());
```

Comment:
move down to its usage

Reply:
I totally forgot about an assertion I wanted to add there. Will fix that.